src/app/core/dialogs/report-item-dialog/report-item-dialog.component.html

```diff
@@ -5,7 +5,7 @@ <h1>{{ type }} melden</h1>
   </div>
   <div class="message">
     <div class="message__intro">Erkläre uns hier kurz, warum du den {{ type }} meldest</div>
-    <textarea class="textarea" rows="5" [(ngModel)]="data.message" name="message" aria-label="message" required></textarea>
+    <textarea class="textarea" rows="5" [(ngModel)]="data.message" placeholder="Inhalt eingeben" onclick="this.placeholder=''" name="message" aria-label="message" required></textarea>
```

Comment:
Ist es nicht untypisch, dass der `placeholder` verschwindet sobald man in das Feld klickt?
Üblicherweise bleibt der doch so lange stehen, bis etwas in das Feld eingegeben wird.

Reply:
genau, das ist das Standardverhalten. Mir gefällt es weniger, deshalb: bei Onfocus ist der Test weg.
Bei onblur ist er ja dann auch wieder da, wenn man nichts eingibt.
Habe das schon auf einigen Formularen so gesehen und  war angenehm überrascht.
Eigentlich finde ich diese Placeholder bisschen doof (wenn das, was man tun soll, genau darüber steht), aber nach langen Debatten mit Henning und Patrick haben wir uns dafür entschieden, Placeholder einzusetzen.

Reply:
OnClick ist nötig, wenn das Feld einen Autofocus hat bzw. es das einzige Eingabefeld auf der Seite ist.
Da würde onfocus sofort wirken und den placeholder löschen.